yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -216,6 +242,41 @@ module vyatta-protocols-frr-bgp-v1 {
 		}
 	}
 
+  grouping network-ipv4-vpn {
+    list network {
+      description "BGP network";
+      configd:help "BGP network";
+      key "addr";
+      leaf addr {
+        description "BGP network";
+        type types:ipv4-prefix {
+          configd:normalize "normalize ipv4-prefix";
+        }
+        configd:help "BGP network";
+      }
+      leaf rd {
+        description "Specify Route Distinguisher";
+        configd:help "Specify Route Distinguisher";
+        configd:pattern-help "ASN:NN or IPAddress:NN";
+        type rt-rd;
+        default "100:1";
+      }
+      leaf label {
```

Comment:
why is this needed ? bgp should allocate the vpn label. The "label vpn export auto" configuration assigns a vrf label ? Why can't this be done instead of using a static label ?